### PR TITLE
For #23498 improve logo content description comments for wallpapers

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -468,7 +468,8 @@
     <string name="wallpaper_tap_to_change_switch_label">Tap logo to change wallpaper</string>
     <!-- Description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
     <string name="wallpaper_tap_to_change_switch_description">Cycle through and update image without leaving the homepage.</string>
-    <!-- Content description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
+    <!-- This is the accessibility content description for the wallpapers functionality. Users are
+    able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
     <string name="wallpaper_logo_content_description">Firefox logo - change the wallpaper, button</string>
```
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:473]: I am wondering why we have `, ` before the button here?

Comment (Amejia481) [reply app/src/main/res/values/strings.xml:473]: We got the text from product, you could read more here https://mozilla-hub.atlassian.net/browse/FNXV2-19315

Comment (Amejia481) [reply app/src/main/res/values/strings.xml:473]: Should we remove it?

Comment (Delphine) [reply app/src/main/res/values/strings.xml:473]: The updated comment looks good to me.
FWIW, concerning the comma, I think I've seen this format used in other accessibility strings.

Comment (Amejia481) [reply app/src/main/res/values/strings.xml:473]: Thanks for the clarification @Delphine!

**Review by Delphine — APPROVED** [review]

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**